CUSTOM_IMAGE.md

```diff
@@ -0,0 +1,76 @@
+Building a Custom Spark Image
+=============================
+
+Overview
+--------
+To build a custom Spark image, with a custom version of Spark or other libraries, 
+such as Hadoop, Python, or R, follow the steps below. This build does the following:
+1. Builds a Docker image containing Spark and other libraries.
+1. Builds a DC/OS package (also called a "universe") based on this Docker image.
```

Comment:
They shouldn't have to build a custom universe.  The spark CLI supports a --docker-image parameter, where they can specify a custom docker image.